Backend/CarRentalApp/CarRentalApp/Services/Data/Tokens/RefreshTokenRepository.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using CarRentalApp.Contexts;
+using CarRentalApp.Models.Entities;
+using Microsoft.EntityFrameworkCore;
+
+namespace CarRentalApp.Services.Data.Tokens
+{
+    public class RefreshTokenRepository
+    {
+        private readonly CarRentalDbContext _authenticationDbContext;
+
+        public RefreshTokenRepository(CarRentalDbContext authenticationDbContext)
+        {
+            _authenticationDbContext = authenticationDbContext;
+        }
+
+        public async Task<RefreshToken> CreateAsync(RefreshToken refreshToken)
+        {
+            _authenticationDbContext.RefreshTokens.Add(refreshToken);
+            await _authenticationDbContext.SaveChangesAsync();
+
+            return refreshToken;
+        }
+
+        public async Task DeleteAsync(RefreshToken refreshToken)
+        {
+            _authenticationDbContext.RefreshTokens.Remove(refreshToken);
+            await _authenticationDbContext.SaveChangesAsync();
+        }
+
+        public async Task DeleteRelatedTokensAsync(Guid userId)
+        {
+            var tokens = await _authenticationDbContext
+                .RefreshTokens
+                .Where(t => t.UserId == userId)
+                .ToListAsync();
```

Comment:
```suggestion
            var tokens = await _authenticationDbContext.RefreshTokens
                .Where(t => t.UserId == userId)
                .ToListAsync();
```
line break usually used before method call